### PR TITLE
CPDP-2322: cap/pr is first return page

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/ChangeRepresenteeContactAddressController.scala [hunk 1]
```diff
@@ -144,7 +144,8 @@ class ChangeRepresenteeContactAddressController @Inject() (
           complete.dateOfDeath,
           Some(newContactDetails),
           hasConfirmedPerson = true,
-          hasConfirmedContactDetails = false
+          hasConfirmedContactDetails = false,
```
Comment (mikail-khan) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/ChangeRepresenteeContactAddressController.scala:147]: why is this needed as part of checking if it is a first return or not?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/ChangeRepresenteeContactAddressController.scala:147]: do you mean `hasConfirmedContactDetails`? That flag isn't required for the first return stuff, that was there before. it tells us whether or not the person has clicked confirm on the cya page where we display contact details that the capacitor or personal rep wants to use in the return

Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/ChangeRepresenteeContactAddressController.scala:147]: ignore that - I haven't woken up yet
